selfdrive/controls/controlsd.py

```diff
@@ -478,25 +474,29 @@ def state_control(self, CS):
     long_plan = self.sm['longitudinalPlan']
 
     actuators = car.CarControl.Actuators.new_message()
+    # Check if actuators are enabled
+    actuators.latActive = self.enabled and not CS.gasPressed
```

Comment:
Disengage on gas logic will be here to keep steering in preEnable/enable

Reply:
Previous active logic was to use the pre-enable state, which right now is only for gas press, so we can use state instead. But that's assuming pre-enable will always allow us to start steering if disengage on gas is false.